index.js

```diff
@@ -244,7 +244,14 @@ export default {
             fromEthAddress = web3in.eth.accounts[0];
           }
           if (obj.fromFullShardKey === undefined || obj.toFullShardKey === undefined) {
-            throw new Error('`fromFullShardKey` and `toFullShardKey` are required');
+            if (obj.fromFullShardId&&obj.fromFullShardId.length>0){
```

Comment:
which on is the right one? fromFullShardId or fromFullShardKey?

Reply:
fromFullShardId  is  in readme  .If not modify readme,use above,and I just committed it again

Reply:
> which on is the right one? fromFullShardId or fromFullShardKey?
ok ,  i have just repaired  it